### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by mayank99**
Updated @itwin/itwinui-css to 0.11.0 which has the tooltip fix.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+`2021-04-29`
+
+### What's new
+
+- **`DropdownMenu` now accepts `popoverProps`**.
+
+### Fixes
+
+- **Fixed image sizing issues in `ErrorPage`**.
+- **Fixed tooltip text blurring issues** through base CSS package.
```
Comment (mayank99) [inline CHANGELOG.md:14]: Should I close #25 now that don't need it? It could still help with something idk

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/itwinui-react",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (mayank99) [inline package.json:3]: I can't decide if this should be 1.2.0 or 1.1.1

The `popoverProps` thing is technically an addition but it's not really a feature, so I went with 1.1.1

File: CHANGELOG.md (hunk 1 above)
Comment (mayank99) [reply CHANGELOG.md:14]: Also @veekeys @bentleyvk can you test if the blurriness is actually gone? I think it looks good on my machine but it's hard to tell

File: package.json (hunk 2 above)
Comment (bentleyvk) [reply package.json:3]: I would say it should be `1.2.0`.

File: CHANGELOG.md (hunk 1 above)
Comment (bentleyvk) [reply CHANGELOG.md:14]: Seems like it is fixed for me.
And about #25 it is now unnecessary and may bring unknown bugs, I would say to close it for now.

Comment (veekeys) [reply CHANGELOG.md:14]: It is fixed, but controlled story does not work for me.
Also, before release, need to fix wizard story, which got broken for some reason.

**Review by veekeys — APPROVED** [review]

**Pull request merged**